wp-content/themes/cpipr/series-single.php

```diff
@@ -193,6 +195,7 @@ function getNextPost($post, $series_map)
 			</div>
 		</div>
 	</div>
+	<?php endif;?>
```

Comment:
Either this should be deindented one tab, or the opening end of the `if` statement, and all lines within the `if` statement, should be indented one tab.

Reply:
ok, done. please check